be/src/vec/columns/column_dictionary.h

```diff
@@ -257,6 +257,9 @@ class ColumnDictionary final : public COWHelper<IColumn, ColumnDictionary<T>> {
     bool is_dict_code_converted() const { return _dict_code_converted; }
 
     MutableColumnPtr convert_to_predicate_column_if_dictionary() override {
+        if (is_dict_sorted() && !is_dict_code_converted()) {
```

Comment:
It seems that `is_dict_sorted()` and `is_dict_code_converted()` are always the same value.

Reply:
Please refer to problem summary.